Assets/Tests/Tests/ConfigurationTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Plugins.CountlySDK.Models;
+using Plugins.CountlySDK;
+using Plugins.CountlySDK.Enums;
+using System.Threading.Tasks;
+
+namespace Tests
+{
+    public class ConfigurationTests
+    {
+        private readonly string _serverUrl = "https://xyz.com/";
+        private readonly string _appKey = "772c091355076ead703f987fee94490";
+
+
+        [Test]
+        public void TestSDKInitialize()
```

Comment:
This should not be a separete test. This is the same test as "TestDefaultConfigValues" just with different asserts. Move these asserts to a different test.